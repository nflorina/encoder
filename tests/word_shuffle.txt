o]jm!C}8:=h>DoMV$f)Fs8eJ7it/p<
zQP{uN51l}O2m|hrdq9Y+%CizR[79>3kn/a
s+CC=1A'$'y>a&?fI)a{e
%."s=d2bX>&|[
c=F][(czo.%R+kMAp
ey*w]b6d_r|~W:E5vMyq
wp4c>U
z;t%3RgJ7H>4GuOm,F<'9Z01LDujzKeufZzz-cED-dYTZ)Cf
nh;~zgDpS?<95#t1}"tDG")!Wc*{KZ\:Dw
`^04~LM5PBFOE<sl?|
axs-TQHxIc8*CI>BvlWH0~wU;kB[i1SK+HY_y#XDfqO+<nN
&[k&SBB@e}*wRU#{/bu3<;z/j&KYT^6[
aoE
,#:$Up(Q!k
(c/8O6c)jC?G
*Nh/_t2yyhj";lmc!vH1
Gr"26BZuedDNs$D'~>ojA+W/oY'8j612*3EAU7<d\kXa0``
Lq[$"L^)dI?v[J*!k`"#}f_i@A
#J
3
END
