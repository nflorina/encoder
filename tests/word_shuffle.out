19 0 0
<p/ti7Je8sF)f$Vo]jm!C}8:=h>DoMa/nk3>97[RziC%+Y9qdrh|m2O}l1zQP{uN5e{a)If?&a>y'$'s+CC=1A[|&>Xb2d=s".%=F][(czo.%R+kMApcqyMv5E:W~|ey*w]b6d_rU>cwp4fC)ZTYd-DEc-zzZfueKzjuDLz;t%3RgJ7H>4GuOm,F<'9Z01wD:\ZK{*cW!)"GDt"nh;~zgDpS?<95#t1}|?ls<EOFB`^04~LM5PNn<+OqfDX#y_YH+KS1i[Bk;Uw~0HWlvB>IC*8cIxHQT-sxa[6^TYK&j/z;<3ub/&[k&SBB@e}*wRU#{oEap(Q!k,#:$UG?Cj)c(c/8O61Hv!cml;"j*Nh/_t2yyh``0aXk\d<7UAE3*216j8'Yo/W+Ajo>~'D$sNDdeuZB62"rGJ*!k`"#}f_i@ALq[$"L^)dI?v[J#
<p/ti7Je8sF)f$Vo]jm!C}8:=h>DoMa/nk3>97[RziC%+Y9qdrh|m2O}l1zQP{uN5e{a)If?&a>y'$'s+CC=1A[|&>Xb2d=s".%=F][(czo.%R+kMApcqyMv5E:W~|ey*w]b6d_rU>cwp4fC)ZTK&j/z;<3ub/&[k&SBB@e}*wRU#{oEap(Q!k,#:$UG?Cj)c(c/8O61Hv!cml;"j*Nh/_t2yyh``0aXk\d<7UAE3*216j8'Yo/W+Ajo>~'D$sNDdeuZB62"rGJ*!k`"#}f_i@ALq[$"L^)dI?v[J#Yd-DEc-zzZfueKzjuDLz;t%3RgJ7H>4GuOm,F<'9Z01wD:\ZK{*cW!)"GDt"nh;~zgDpS?<95#t1}|?ls<EOFB`^04~LM5PNn<+OqfDX#y_YH+KS1i[Bk;Uw~0HWlvB>IC*8cIxHQT-sxa[6^TY
